Cookbook/Technical-Documents/LabelsInPRs.md

```diff
@@ -0,0 +1,118 @@
+- <span style="background-color:#fcc1ba; color:black; font-weight:600">Hotfix 🆘</span> - The PR has priority to be reviewed because it has a hot fix targeted to the current release branch.
+
+### <a name="extra"></a>Extra
+These tags are used to give extra context to the PR.
+
+- <span style="background-color:#ff69b4; color:black; font-weight:600">Bug 🐛</span> - The PR fixes a bug.
+- <span style="background-color:#c79ee2; color:black; font-weight:600">Housekeeping 🏡</span> - The PR includes some work that won't change any functionality - it just removes clutter from the project.
+- <span style="background-color:#bfdadc; color:black; font-weight:600">Refactoring 🏗️</span> - The PR has refactoring work. Refactoring work is usually work that involves rewritting a part of the project that had technical debt. This work usually doesn't have any visual changes (UI/UX changes).
+- <span style="background-color:#bfdadc; color:white; font-weight:600">Trivial 👶</span> - The PR takes 1-2 minutes to review (an example can be a PR that only involves updading localizable strings).
+
+## 2. The rules
+
+A PR should always present at least 2 labels. One label should represent the area of the project and the other should represent the stage of the PR.
+
+There are a few rules we should consider when applying labels to a PR.
+
+
+### [**Product verticals**](#product-verticals), [**Project horizontals**](#project-horizontals) and [**Branding**](#branding)
+
+A PR should always contain at least one label that belongs to these groups. Multiple labels from these groups can be added to a PR because a piece of work can have impact in more than one area.
+
+### [**Stage**](#stage)
+
+In this group a mixed of rules are applied. 
+A PR can:
+
+- be either [ready for review](#ready_review) or still [in progress](#in_progress). These are mutually exclusive. It means the labels representing both these stages can't be present in the same instant.
+- have [one reviewer](#needs_reviewer) label only if it is [ready for review](#ready_review).
+- have [merge](#merge) label only if it is [ready for review](#ready_review), has two approvals and no changes requested.
+
+### [**Status**](#status)
+
+A PR can have one or both of these labels dependending on the blocking chain. However, it is very unusual and not ideal that a PR is in a situation where it needs both labels at the same time.
+
+There is some specific information that is mandatory to be present in the PR description when using:
+
+- [Blocked](#blocked) - Describe the reason why it is blocked. If it is being blocked by another PR or JIRA Ticket, these should be mentioned.
+- [Blocking other PRs](#blocking) - Describe the reason why it is blocking other PRs. It should mentioned which PRs are being blocked by it.
+
+### [**Emergency**](#emergency)
+
+A PR can only have one emergency label.
+
+### [**Extra**](#extra)
+
+A PR can have one or more extra labels at the same time.
```

Comment:
Why is that? I mean one could imagine having a PR that is Trivial + Housekeeping if it just removes a file that is no longer used or similar...

Reply:
I think you understood it wrong 😛 
It says
> A PR can have one or more extra labels at the same time.

Means it can have Trivial + Housekeeping if you like to.

Reply:
🤦‍♂ I need coffee. ☕️